CONTRIBUTING.md

```diff
@@ -211,21 +212,21 @@ replace `<GOOS>/<GOARCH>` with your particular operating system and compilation
        -profile=<cpu or memory> \
        -f secretless.yml
    ```
-   Note: the location of the binary might vary across different OS
+   *Note*: the location of the binary might vary across different OS
```

Comment:
Ditto on the styling + imperative language

Reply:
this is not in imperative language like the below statement?